src/test/java/BaseTest.java

```diff
@@ -40,13 +40,35 @@ public void launchBrowser(String BaseURL) {
         chromeOptions.addArguments("--disable-notifications");
 
         driver = new ChromeDriver(chromeOptions);
+        driver.manage().window().maximize();
         actions = new Actions(driver);
-        wait = new WebDriverWait(driver, Duration.ofSeconds(50));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
         url = BaseURL;
         driver.get(url);
-
     }
 
+   // @BeforeTest
+   // public void login() {
+       // By emailLocator = By.xpath("//input[@placeholder='Email Address']");
+        //wait.until(ExpectedConditions.elementToBeClickable(emailLocator));
+       // WebElement emailField = driver.findElement(emailLocator);
+       // emailField.click();
+       // emailField.sendKeys("holostenco.yuliya@gmail.com");
+
+       // By passwordLocator = By.xpath("//input[@type='password']");
+       // wait.until(ExpectedConditions.elementToBeClickable(passwordLocator));
+       // WebElement passwordField = driver.findElement(passwordLocator);
+        //passwordField.click();
+        //passwordField.sendKeys("te$t$tudent");
+
+       // By submitButtonLocator = By.xpath("//button[@type='submit']");
+       // wait.until(ExpectedConditions.elementToBeClickable(submitButtonLocator));
+       // driver.findElement(submitButtonLocator).click();
```

Comment:
Please, avoid any comments that are likely to get out of date as the code evolves.